### PR TITLE
AA/AE_TestCase2

### PR DESCRIPTION
**Pull request by afaabb**

### CODE REVIEW

File: cypress/cypress/e2e/AE_Tests.cy.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ describe("AE Tests", () => {
   const username = "name";
   const testsNeedingAccount = ["AE_TestCase2"];
 
-  beforeEach(function() {
+  beforeEach(function () {
```
Comment (teachandlearn999) [inline cypress/cypress/e2e/AE_Tests.cy.js:8]: great job

File: cypress/cypress/e2e/AE_Tests.cy.js [hunk 2]
```diff
@@ -35,51 +35,69 @@ describe("AE Tests", () => {
...
-})
-  
-  
-  it("AE_TestCase2", () => {
+  it("AE_TestCase1", () => {
+    cy.visit("https://automationexercise.com/");
+    cy.get('.nav a[href="/"]').should("have.css", "color", "rgb(255, 165, 0)");
+    cy.get('.nav a[href="/login"]').click();
+    cy.get(".signup-form h2")
+      .should("be.visible")
```
Comment (teachandlearn999) [inline cypress/cypress/e2e/AE_Tests.cy.js:43]: this should be on one line

File: cypress/cypress/e2e/AE_Tests.cy.js (hunk 1 above)
Comment (Stavolg) [reply cypress/cypress/e2e/AE_Tests.cy.js:8]: extra space

**Review by teachandlearn999 — CHANGES_REQUESTED** [review]

**Review by Stavolg — APPROVED** [review]

**Review by teachandlearn999 — APPROVED** [review]

**Review by Stavolg — APPROVED** [review]

File: cypress/cypress/e2e/AE_Tests.cy.js [hunk 3]
```diff
@@ -35,51 +35,69 @@ describe("AE Tests", () => {
...
+      .should("have.text", "New User Signup!");
+    cy.get('.signup-form input[name="name"]').type(username);
+    cy.get('.signup-form input[name="email"]').type(
+      `email_${Date.now()}@gmail.com`
+    );
+    cy.get(".signup-form button").click();
+    cy.get("#id_gender2").click();
+    cy.get("#password").type(password);
+    cy.get("#days").select("1");
+    cy.get("#months").select("1");
```
Comment (teachandlearn999) [inline cypress/cypress/e2e/AE_Tests.cy.js:53]: plusik

**Review by teachandlearn999 — APPROVED** [review]

**Pull request merged**